### PR TITLE
Remove config value ANALYSIS_LOAD

### PR DESCRIPTION
**Pull request by joakim-hove**
All analysis modules are now internal - `ANALYSIS_LOAD` config keyword can not do anything

### CODE REVIEW

**Comment by ertomatic** [comment]
Can one of the admins verify this patch?

**Comment by sondreso** [comment]
Jenkins ok to test

**Comment by oyvindeide** [comment]
Note that this closes: #2395

File: docs/rst/manual/reference/configuration/keywords.rst [hunk 1]
```diff
@@ -1411,12 +1400,10 @@ to load, select and modify the analysis modules are documented here.
 
         ::
 
-                ANALYSIS_LOAD   A1  analysis.so
                 ANALYISIS_COPY  A1  A2
```
Comment (oyvindeide) [inline docs/rst/manual/reference/configuration/keywords.rst:1403]: Outside the diff of this PR, but perhaps fix this typo while you are here?

Comment (joakim-hove) [reply docs/rst/manual/reference/configuration/keywords.rst]: Done

**Comment by joakim-hove** [comment]
test this please

**Comment by joakim-hove** [comment]
test this please

**Comment by joakim-hove** [comment]
test this please

File: docs/rst/manual/reference/configuration/keywords.rst [hunk 2]
```diff
@@ -1375,9 +1367,6 @@ to load, select and modify the analysis modules are documented here.
 
                 ANALYSIS_SELECT ANAME
 
-        Here ANAME is the name you have assigned to the module when loading it with
```
Comment (BjarneHerland) [inline docs/rst/manual/reference/configuration/keywords.rst:1378]: Suggest e.g.
```
        Here ANAME is one of `IES` and `STD_ENKF` which currently are the two available
        analysis modules.
```

File: docs/rst/manual/reference/configuration/keywords.rst [hunk 3]
```diff
@@ -1404,12 +1393,10 @@ to load, select and modify the analysis modules are documented here.
...
         ::
 
-                ANALYSIS_LOAD   A1  analysis.so
-                ANALYISIS_COPY  A1  A2
+                ANALYSIS_COPY  A1  A2
 
-        We load a module analysis.so and assign the name A1; then we copy A1 -> A2.
-        The module A1 and A2 are now 100% identical. We then set the truncation to two
-        different values:
+        We copy A1 -> A2. The module A1 and A2 are now 100% identical. We then
```
Comment (BjarneHerland) [inline docs/rst/manual/reference/configuration/keywords.rst:1398]: Suggest e.g.
```
        A1 can currently be one of `IES` and `STD_ENKF`.
        The modules A1 and A2 are now 100% identical. We then set the truncation to two
        different values:

```

**Review by BjarneHerland — COMMENTED** [review]
Forgot these?

https://github.com/equinor/ert/blob/d194fa03af5c6aa06882f9a4f6e7cd9a4a2976ab/test-data/local/simple_config/analysis_config#L7-L8

File: docs/rst/manual/reference/configuration/keywords.rst (hunk 3 above)
Comment (joakim-hove) [reply docs/rst/manual/reference/configuration/keywords.rst]: OK

File: docs/rst/manual/reference/configuration/keywords.rst (hunk 2 above)
Comment (joakim-hove) [reply docs/rst/manual/reference/configuration/keywords.rst:1378]: OK

**Comment by joakim-hove** [comment]
test this please

**Comment by oyvindeide** [comment]
Jenkins: test ert please

**Comment by oyvindeide** [comment]
Can you do a rebase? There are some conflicts, other than that it looks like everything is addressed @BjarneHerland?

**Comment by joakim-hove** [comment]
> Can you do a rebase?

Done!

**Review by BjarneHerland — APPROVED** [review]
I was pretty sure I had already approved this one...  LGTM, but probably needs to be rebased (again) and pass tests before it technically can be merged

**Pull request merged**